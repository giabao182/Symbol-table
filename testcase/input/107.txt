INSERT a number
INSERT b number
ASSIGN a b