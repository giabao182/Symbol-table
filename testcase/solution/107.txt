success
Invalid: INSERT b numberASSIGN a b