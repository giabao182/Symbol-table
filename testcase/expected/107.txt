Undeclared: ASSIGN x 4